pkg/rhcos/powervs_regions.go

```diff
@@ -0,0 +1,86 @@
+package rhcos
+
+// Since there is no API to query these, we have to hard-code them here.
+
```

Comment:
this should come from the regions that RHCOS is pushed to which is parsed from the stream metadata.similar to how AMI regions are done through the generate script. 

But the question here is - are these the regions where the bootimage will be imported to ?

Reply:
ok, so we'll probably have to come up with a kind of hybrid approach then, or move this somewhere outside the rhcos purview, and combine what's there with the stream metadata. the regions at the top level are the Power VS regions -- not the IBM Cloud DC/COS regions the bootimage will be imported to.